ddtrace/monkey.py

```diff
@@ -54,6 +54,9 @@
     "falcon": False,
     "pylons": False,
     "pyramid": False,
+
+    # Standard library modules off by default
+    "logging": False,
```

Comment:
```suggestion
    'logging': False,
```

Reply:
also, I don't think we should include in here like this.

Mainly because enabling via `DATADOG_PATCH_MODULES=logging:true` seems really weird to me, especially since we aren't tracing the `logging` library.

I think moving `patch_logging` into `monkey.py` so people can do `from ddtrace.monkey import patch_logging; patch_logging()` manually might be better?